Dockerfile-alpine

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.14 as certs
+FROM alpine:3.14
```

Comment:
3.14 has been deprecated for quite some time, 3.18 is the latest and greatest

![image](https://github.com/yannh/kubeconform/assets/1580956/e91634a5-ad20-41c7-80ac-3746de344d67)

Reply:
I create a PR for updating the alpine base image dependency as well, https://github.com/yannh/kubeconform/pull/203